implementation/README.md

```diff
@@ -30,14 +19,12 @@ npm run compile
 
 ```sh
 # Show issues
-npm run js:lint
-npm run sol:lint
+npm run lint
 
 # Automatically fix issues
-npm run js:lint:fix
-npm run sol:lint:fix
+npm run lint:fix
 ```
 
-Eslint errors can be disabled using a comment on the previous line. For example, 
-to disable linter errors for the 'no-unused-vars' rule: 
-`// eslint-disable-next-line no-unused-vars`.
\ No newline at end of file
+Eslint errors can be disabled using a comment on the previous line. For example,
+to disable linter errors for the 'no-unused-vars' rule:
+`// eslint-disable-next-line no-unused-vars`.
```

Comment:
I would love to delete this altogether. Though occasionally necessary, lint skips are IMO not a great idea, so recommending them in the README seems like an anti-pattern.

Reply:
Agreed, all I did was remove trailing whitespace though 😂

I'll snip it